onnxruntime/core/providers/ngraph/ngraph_custom_op.cc

```diff
@@ -186,6 +207,7 @@ Status NGRAPHCustomOp::Compute(const OrtCustomOpApi* api, OrtKernelContext* cont
   }
 
   // Run the graph through nGraph.
+  std::lock_guard<std::mutex> lock(compute_lock_);
```

Comment:
I think Initialize() (import and compilation) may not be thread safe and that is why the lock was held at the beginning of Compute() previously? 
What about ng_backend_->create_tensor() ? is it thread safe?